src/main/java/com/couchbase/client/spring/cache/CouchbaseCacheManager.java

```diff
@@ -38,27 +38,36 @@
 public class CouchbaseCacheManager extends AbstractCacheManager {
 
   private CacheBuilder defaultCacheBuilder;
-  private final Collection<CouchbaseCache> initialCaches;
+  private final Map<String, CacheBuilder> initialCaches;
 
   /**
-   * Construct a {@link CacheManager} knowing about a predetermined set of caches at construction. The caches are
-   * all prepared using the provided {@link CacheBuilder} (at least specifying a backing {@link Bucket}).
+   * Construct a {@link CacheManager} with a "template" {@link CacheBuilder} (at least specifying a backing
+   * {@link Bucket}).
```

Comment:
I am not sure that's very useful to copy/paste the constraint. You can't create a `CacheBuilder` without a `Bucket` so I wouldn't say it here again.